pandas/io/sql.py

```diff
@@ -1235,6 +1236,14 @@ def _get_dtype(self, sqltype):
             Integer,
         )
 
+        # GH34988 - Workaround Oracle Dialect NUMBER defaulting to int
```

Comment:
I think can remove this comment - nice idea but if we have snippets like this from every PR it gets a bit unmanageable. I think the code is clear enough